cmd/scale.go

```diff
@@ -188,11 +194,17 @@ func (sc *scaleCmd) validate(cmd *cobra.Command, args []string) {
 
 	nameSuffixParam := templateParameters["nameSuffix"].(map[string]interface{})
 	sc.nameSuffix = nameSuffixParam["defaultValue"].(string)
-	log.Infoln(fmt.Sprintf("Name suffix: %s", sc.nameSuffix))
+	log.Infoln("Name suffix: %s", sc.nameSuffix)
+	return nil
 }
 
 func (sc *scaleCmd) run(cmd *cobra.Command, args []string) error {
-	sc.validate(cmd, args)
+	if err := sc.validate(cmd); err != nil {
+		log.Fatalln("failed to validate scale command: %s", err.Error())
```

Comment:
we don't want to return here?

Reply:
we do... I left the run function for another day because it has a ton of those but I'll fix that one since I added it

Reply:
Nevermind I just fixed all of them